.travis.yml

```
@@ -52,7 +54,11 @@ script:
  - mkdir build
  - cd build
  - cp ../AMBuildScript ../../
- - python2 ../configure.py -s sdk2013 --hl2sdk-root ../assets --sm-path ../assets/sourcemod-central --mms-path ../assets/mmsource-central --steamworks-path ../assets/SteamworksSDK 
+ - env
+ - eval "$(exec /usr/bin/env -i "${SHELL}" -l -c "export")"
+ - env
+ - (uname -a | grep Msys) && powershell "cmd /c "\"$VS140COMNTOOLS/vcvars32.bat\" && $SNAKEWRAP ../configure.py -s sdk2013 --hl2sdk-root ../assets --sm-path ../assets/sourcemod-central --mms-path ../assets/mmsource-central --steamworks-path ../assets/SteamworksSDK""
```

```suggestion
 - (uname -a | grep Msys) && powershell.exe "cmd /c 'call \`"${ENV:VS140COMNTOOLS}\vsvars32.bat\`" && ${ENV:SNAKEWRAP} ../configure.py -s sdk2013 --hl2sdk-root ../assets --sm-path ../assets/sourcemod-central --mms-path ../assets/mmsource-central --steamworks-path ../assets/SteamworksSDK'"
```

Here's output of running this suggestion on my system (without cloning the repository):
```
PS C:\Users\lollo> powershell.exe "cmd /c 'call \`"${ENV:VS140COMNTOOLS}\vsvars32.bat\`" && ${ENV:SNAKEWRAP} ../configure.py -s sdk2013 --hl2sdk-root ../assets --sm-path ../assets/sourcemod-central --mms-path ../assets/mmsource-central --steamworks-path ../assets/SteamworksSDK'"
python: can't open file '../configure.py': [Errno 2] No such file or directory
```